tests/server/server-getTags-002.phpt

```diff
@@ -12,20 +12,27 @@ $manager = new MongoDB\Driver\Manager(URI);
 $command = new MongoDB\Driver\Command(['ping' => 1]);
 $manager->executeCommand(DATABASE_NAME, $command);
 
-foreach ($manager->getServers() as $server) {
-    $tags = $server->getTags();
-    echo "dc: ", array_key_exists('dc', $tags) ? $tags['dc'] : 'not set', "\n";
-    echo "ordinal: ", array_key_exists('ordinal', $tags) ? $tags['ordinal'] : 'not set', "\n";
+function assertSomeServerHasTags(array $servers, array $expectedTags) {
+    foreach ($servers as $server) {
+        if ($expectedTags == $server->getTags()) {
```

Comment:
If `getTags()` returns an array, is there any reason we can't do a strict equality check here?

Both are going to enforce the same order AFAIK, and the associative array itself just has string keys and values (no concern with mismatched types).

Reply:
I changed to a strict comparison, which then caused tests to fail again. With some debug output, I was able to capture the following diff:
```patch
003+ No server has tags: {"ordinal":"two","dc":"pa"}
004+ Server tags: {"localhost:27017":{"ordinal":"one","dc":"ny"},"localhost:27018":{"dc":"pa","ordinal":"two"},"localhost:27019":[]}
003- Found server with tags: {"ordinal":"two","dc":"pa"}
```

As you can see, the tags returned for the second server has a different order from the first, which causes a strict equality check to fail. I've added a comment to clarify why we're using `==` in the test for future reference.